src/datatypes/preset.js

```diff
@@ -0,0 +1,168 @@
+/**
+ * ATTENTION. This code was AUTO GENERATED by schema2typebox.
+ * While I don't know your use case, there is a high chance that direct changes
+ * to this file get lost. Consider making changes to the underlying JSON schema
+ * you use to generate this file instead. The default file is called
+ * "schema.json", perhaps have a look there! :]
+ */
+import { Type } from '@sinclair/typebox'
+
+export const Preset = Type.Object(
+  {
+    docId: Type.String({
+      description: 'Hex-encoded 32-byte buffer',
+      minLength: 1,
+    }),
+    versionId: Type.String({
+      description:
+        "core discovery id (hex-encoded 32-byte buffer) and core index number, separated by '/'",
+      minLength: 1,
+    }),
+    originalVersionId: Type.String({
+      description:
+        'Version ID of the original version of this document. For the original version, matches `versionId`.',
+      minLength: 1,
+    }),
+    schemaName: Type.Literal('preset', { description: 'Must be `preset`' }),
+    createdAt: Type.String({
+      description:
+        'RFC3339-formatted datetime of when the first version of the element was created',
+      format: 'date-time',
+    }),
+    updatedAt: Type.String({
+      description:
+        'RFC3339-formatted datetime of when this version of the element was created',
+      format: 'date-time',
+    }),
+    links: Type.Array(Type.String(), {
+      description:
+        "Version ids of the previous document versions this one is replacing. Each link is id (hex-encoded 32 byte buffer) and index number, separated by '/'",
+      uniqueItems: true,
+    }),
+    deleted: Type.Boolean({
+      description: 'Indicates whether the document has been deleted',
+    }),
+    name: Type.String({
+      description: 'Name for the feature in default language.',
+    }),
```

Comment:
Is there a sanitized preset name returned anywhere (e.g. "My CoMapeo _Category!" -> `my_comapeo__category_`), or is the expectation that this be handled downstream if needed?

(Fine either way, just asking for my understanding.)

Reply:
No sanitizing happening, because the requirements for how any value needs to be sanitized are dependent on how the data is being consumed. Yes, this is the category name.